README.md

```diff
@@ -1,4 +1,750 @@
+In the event of a non-200 error return code, the transaction ID is also returned in the JSON error response as `incident_id`.
+
+
+## Using the REST API to administer Kafka topics
+---
+
+To run the example :-
+
+Compile the code.
+```sh
+go build -o example
+```
+Or simply 
+```sh
+make build
+```
+
+Set the required environment variables
+```sh
+# Set you API KEY.
+export API_KEY="abc123456789"
+
+# Set the Admin Endpoint to point to your cluster.
+export KAFKA_ADMIN_URL="https://xyzclustername.svc01.region.eventstreams.test.cloud.ibm.com"
+
+```
+
+Run the example
+```sh
+./example 
+```
+
+## REST API 
+---
+The following sections explain how the REST API works with examples.
+
+### Code Setup
+
+	// Code Setup
+	import (
+		"fmt"
+		"net/http"
+		"os"
+	
+		"github.com/IBM/eventstreams-go-sdk/pkg/adminrestv1"
+		"github.com/IBM/go-sdk-core/v4/core"
+	)
+	
+	// End Code Setup
+
+
+
+### Authentication
+---
+Use one of the following methods to authenticate:
+
+- To authenticate using Basic Auth:
+  Place these values into the Authorization header of the HTTP request in the form Basic <credentials> 
+  where <credentials> is the username and password joined by a single colon `:` base64 encoded. 
+  For example:
+  ```sh
+  echo -n "token:<APIKEY>" | base64
+  ```
+
+- To authenticate using a bearer token:
+  To obtain your token using the IBM Cloud CLI, first log in to IBM Cloud, then run the following command:
+  ```
+  ibmcloud iam oauth-tokens
+  ```
+  Place this token in the Authorization header of the HTTP request in the form Bearer. Both API key or JWT tokens are supported.
+
+- To authenticate directly using the api_key:
+  Place the key directly as the value of the X-Auth-Token HTTP header.
+
+#### Example
+
+Here's an example of how to create the authenticator using either an API key or a BEARER_TOKEN
+
+		// Create Authenticator
+		var authenticator core.Authenticator
+	
+		if apiKey != "" {
+			var err error
+			// Create an Basic IAM authenticator.
+			authenticator, err = core.NewBasicAuthenticator("token", apiKey)
+			if err != nil {
+				fmt.Printf("failed to create new basic authenticator: %s\n", err.Error())
+				os.Exit(1)
+			}
+		} else {
+			var err error
+			// Create an IAM Bearer Token authenticator.
+			authenticator, err = core.NewBearerTokenAuthenticator(bearerToken)
+			if err != nil {
+				fmt.Printf("failed to create new bearer token authenticator: %s\n", err.Error())
+				os.Exit(1)
+			}
+		}
+		// End Authenticator
+
+
+
+
+### Creating a client for the Admin REST API.
+---
+Create a new service object.
+
+		// Create Service
+		serviceAPI, serviceErr := adminrestv1.NewAdminrestV1(&adminrestv1.AdminrestV1Options{
+			URL:           URL,
+			Authenticator: authenticator,
+		})
+		// End Create Service
+
+
+
+### Creating a Kafka topic
+---
+To create a Kafka topic the admin REST SDK issues a POST request to the /admin/topics path. 
+The body of the request contains a JSON document, for example:
+{
```

Comment:
This is not formatted correctly, see https://github.com/IBM/eventstreams-go-sdk/blob/2b01f03571341cbe62e1cce2e993d76645968d4b/README.md#creating-a-kafka-topic
It needs to be wrapped in a code block